### PR TITLE
Added storybook environment to the project

### PR DESCRIPTION
**Pull request by Joemonmathew**

### CODE REVIEW

**Comment by Joemonmathew** [comment]
I have added storybook environment using following steps
installed storybook by command:
 'npx sb init'
after installing storybook to the project we can run it using 
'yarn stroybook'

**Comment by ajayesivan** [comment]
Screenshot 2023-08-11 at 13 46 04

We can remove this example stories

File: src/stories/Button.jsx
Comment (ajayesivan) [inline src/stories/Button.jsx:1]: You can remove all this example stories

Comment (ajayesivan) [reply src/stories/Button.jsx:1]: We will do this later as a separate issue.

**Review by ajayesivan — APPROVED** [review]
LGTM! 
Followup issue required,
1. #36 
2. #37

**Pull request merged**